src/main/java/org/libvirt/Error.java

```diff
@@ -18,113 +18,209 @@
      * {@code values.length}.
      *
      * @return n-th item of {@code values} when {@code n <
-     *          values.length}, otherwise the last item of {@code values}.
+     * values.length}, otherwise the last item of {@code values}.
```

Comment:
I would bring `values.length},` to the previous line and break the line after the comma.